File: classfields/tests/example/ra/RequestSpecBuilders.java
package ru.auto.tests.example.ra;

import io.restassured.builder.RequestSpecBuilder;

import java.util.function.Consumer;

import static com.google.common.base.Functions.identity;

public class RequestSpecBuilders {

    private RequestSpecBuilders() {
    }

    public static Consumer<RequestSpecBuilder> defaultSpec() {
        return req -> identity();
    }
}

File: classfields/tests-core/src/main/scala/ru/auto/tests/ra/RequestSpecBuilders.scala
package ru.auto.tests.ra

import io.restassured.builder.RequestSpecBuilder
import java.util.function.Consumer

import io.restassured.http.ContentType

object RequestSpecBuilders {

  def defaultSpec: Consumer[RequestSpecBuilder] =
    (req: RequestSpecBuilder) => req

  def acceptText: Consumer[RequestSpecBuilder] = { (req: RequestSpecBuilder) =>
    req.setAccept(ContentType.TEXT)
  }
}
